_posts/2020-11-09-golang-style-guide.adoc

```diff
@@ -0,0 +1,105 @@
+---
+title: Go Style Guide
+categories: Styleguides
+image: /assets/img/guides/go-logo-blue.png
+excerpt: The style guide for the Go programming language that is used by all Gruntwork-owned code repositories that contain Go code.
+tags: ["go", "golang", "code", "style"]
+cloud: ["styleguides"]
```

Comment:
As mentioned above, I think we should actually mark this as all clouds to avoid overloading the cloud filter.